File: src/main/java/ar/edu/unq/spring/modelo/clasesDePersonajes/Guerrero.java
package ar.edu.unq.spring.modelo.clasesDePersonajes;

import ar.edu.unq.spring.modelo.Personaje;
import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class Guerrero extends Personaje {

    private int fuerza = 0;

    public Guerrero() {}

    public Guerrero(String nombre) {
        this.setNombre(nombre);
    }
}

File: src/main/kotlin/ar/edu/unq/spring/modelo/clasesDePersonajes/Guerrero.kt
package ar.edu.unq.spring.modelo.clasesDePersonajes

import ar.edu.unq.spring.modelo.Personaje
import javax.persistence.Entity

@Entity
class Guerrero() : Personaje() {
    var fuerza: Int = 0

    constructor(nombre: String) : this() {
        this.nombre = nombre
    }

}
